conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/CastingRuleSuite.scala

```diff
@@ -125,4 +127,60 @@ class CastingRuleSuite extends FunSuite with SparkTestBase with LoggerTestBase{
 
   }
 
+  test("Test Casting rule ok handling conversion from 'long' to 'string'") {
+    RuleValidators.validateTypeCompatibility(ruleName, columnName, LongType, "string")
+  }
+
+  test("Test Casting rule ok handling conversion from 'string' to 'long'") {
+    RuleValidators.validateTypeCompatibility(ruleName, columnName, StringType, "long")
+  }
+
+  test("Test Casting rule ok handling conversion from 'string' to 'date'") {
+    RuleValidators.validateTypeCompatibility(ruleName, columnName, StringType, "date")
+  }
+
+  test("Test Casting rule ok handling conversion from 'boolean' to 'decimal(10,2)'") {
+    RuleValidators.validateTypeCompatibility(ruleName, columnName, BooleanType, "decimal(10,2)")
+  }
+
+  test("Test Casting rule ok handling conversion from 'long' to 'timestamp'") {
+    RuleValidators.validateTypeCompatibility(ruleName, columnName, LongType, "timestamp")
```

Comment:
Perhaps a comment explaining how the conversion works.

Reply:
Do you mean to explain that long is expected to represent seconds since epoch?

Reply:
Added

Reply:
Yup, thank you. 👍